module/metrics/namespaces.go

```diff
@@ -40,8 +40,9 @@ const (
 
 // Consensus subsystems represent the different components of the consensus algorithm.
 const (
-	subsystemCompliance = "compliance"
-	subsystemHotstuff   = "hotstuff"
+	subsystemCompliance  = "compliance"
+	subsystemHotstuff    = "hotstuff"
+	subsystemMatchEngine = "match"
```

Comment:
The consensus node was using the `subsystemMatchEngine` from the verification node domain, now that we no longer have a match engine, this subsystem label is moved up to the consensus domain.